docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -43,7 +61,22 @@ const hmsActions = hms.getHMSActions();
 -   `Peer`: A participant in the video call. You are the `local peer` while others are `remote peers`.
 -   `Track`: Media. There are two types of track a peer can have - audio and video.
 
-## Joining a room
+### Initialize the HTML elements
+Now, let's initialize all the HTML elements required such as tile for local and remote peers, join and leave button, mute and unmute button, etc.
+
+```jsx:script.js
+// HTML elements
+const form = document.getElementById("join");
+const joinBtn = document.getElementById("join-btn");
+const conference = document.getElementById("conference");
+const peersContainer = document.getElementById("peers-container");
+const leaveBtn = document.getElementById("leave-btn");
+const muteAud = document.getElementById("mute-aud");
+const muteVid = document.getElementById("mute-vid");
+const controls = document.getElementById("controls");
+```
+
+### Join/leave a room
 
 To join a room (a video call), we need to call the `join` method on `hmsActions` and it requires us to pass a `config` object.
```

Comment:
can we point to the docs for this method in the API reference?

Reply:
we can use this quickstart as an opportunity to train the user to use the reference section